spec/factories/posts.rb

```diff
@@ -3,21 +3,24 @@
     title  "Blog Post Title Here"
     body   "A simple blog post"
     status "published"
+    category "Ruby"
 
     factory :blog do
       title "The Booming field of Programming"
       body  "Web development, and programming in general, is a booming field.
-             However not enough college bound young adults are choosing to 
+             However not enough college bound young adults are choosing to
              persue this awesome field of study."
       status "published"
+      category "Programming"
     end
 
     factory :ssh do
       title "Passwordless login with ssh"
-      body   "To be able to login to your remote over ssh, without using 
+      body   "To be able to login to your remote over ssh, without using
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.